scripts/postbuild.sh

```diff
@@ -7,3 +7,17 @@ cp -a ../../../click/* $@
 # Build axolotl-web
 echo "update translations and build axolotl-web $@"
 cd ../../../axolotl-web&&npm run translate &&npm run build && mkdir $@/axolotl-web&&cp dist $@/axolotl-web/ -r
+
+echo $ARCH
+if [ $ARCH == "arm64" ]
+then
+    [ -z '$GOPATH' ] && cp /github/workspace/go/src/github.com/nanu-c/zkgroup/lib/libzkgroup_linux_aarch64.so $@/lib/libzkgroup_linux_aarch64.so; 
+    cp $GOPATH/src/github.com/nanu-c/zkgroup/lib/libzkgroup_linux_aarch64.so $@/lib/libzkgroup_linux_aarch64.so ||true; 
+    rm ${INSTALL_DIR}/\\$GITHUB_WORKSPACE||true
```

Comment:
Is there any reason why we are only escaping INSTALL_DIR with ${}, and not GITHUB_WORKSPACE?

As far as I understand, best practice is to always escape all variables with ${}.

Reply:
This was no specific reason.

Reply:
Lets see, maybe it's not needed anymore.